src/test/java/edu/jsu/mcis/ConverterTest.java

```
@@ -0,0 +1,48 @@
+package edu.jsu.mcis;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.equalTo;
+
+public class ConverterTest {
+
+    private String csvString;
+    private String jsonString;
+
+    @Before
+    public void setUp() throws IOException {
```

Note that here we're simply throwing, rather than catching and printing a stack trace.  We'd rather fail fast in the setup when a file is inaccessible, instead of actually running the assertions and hitting NPE's.